map_it_test.go

```diff
@@ -1057,6 +1057,40 @@ func TestMap_SetTTL(t *testing.T) {
 	})
 }
 
+func TestMap_SetTTLAffected(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx := context.Background()
+		testcases := []struct {
+			key        string
+			isEffected bool
+			expectErr  bool
+		}{
+			{
+				key:        "happy path",
+				isEffected: true,
+			},
+			{
+				key:        "setTTL on non-existing key",
+				isEffected: false,
+				expectErr:  false,
+			},
+			{
+				key:        "setTTL on already expired key",
+				isEffected: false,
+				expectErr:  false,
+			},
+		}
+		_ = it.MustValue(m.Put(ctx, "happy path", "someValue"))
+		_ = it.MustValue(m.PutWithTTL(ctx, "setTTL on already expired key", "someValue", time.Millisecond))
```

Comment:
`setTTL` => `PutWithTTL`?

Reply:
It is just to populate the map. Actual test case is in the for loop body.
If you mean something else can you explain?

Reply:
OK, that key is a bit confusing. How about renaming it something else, maybe something random?

Reply:
Thanks, fixed as suggested!